contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Governable } from "../governance/Governable.sol";
+import { IStrategy } from "../interfaces/IStrategy.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract OETHHarvesterSimple is Governable {
+    ////////////////////////////////////////////////////
+    /// --- STORAGE
+    ////////////////////////////////////////////////////
+    address public strategist;
+    mapping(address => bool) public supportedStrategies;
+
+    ////////////////////////////////////////////////////
+    /// --- EVENTS
+    ////////////////////////////////////////////////////
+    event Harvested(address token, uint256 amount);
+    event StrategistUpdated(address strategist);
+    event SupportedStrategyUpdated(address strategy, bool status);
+
+    ////////////////////////////////////////////////////
+    /// --- MODIFIERS
+    ////////////////////////////////////////////////////
+    modifier onlyStrategist() {
+        require(msg.sender == strategist || isGovernor(), "Not strategist");
+        _;
+    }
+
+    ////////////////////////////////////////////////////
+    /// --- CONSTRUCTOR
+    ////////////////////////////////////////////////////
+    constructor(address _governor, address _strategist) {
+        require(_strategist != address(0), "Invalid strategist");
+        strategist = _strategist;
```

Comment:
Need an event on this initial strategist set. Maybe just make it into an internal method that does the validation and the event, and have the internal used by the public method.

Reply:
Changed in this commit: [00766a9](https://github.com/OriginProtocol/origin-dollar/pull/2333/commits/00766a9838c67a6742a7470e93fe1dd0178db50b).
However, in the `constructor`, the governor is set but doesn't emit an event. Should we emit an event here, too?

Reply:
Good catch. We should use `_changeGovernor()` instead which does emit an event. Confusing naming.

Reply:
Done in this commit: [534ce82](https://github.com/OriginProtocol/origin-dollar/pull/2333/commits/534ce82b6b1adc56e019447d2ab1edd9254838b8)